src/test/ts/commands/StatusTest.ts

```diff
@@ -33,32 +33,50 @@ const check = async (dir: string, expected: Object) => {
 };
 
 describe('Status', () => {
-  it('shows status for main', async () => {
+  it('shows status for main branch in preRelease state', async () => {
     const { dir, git } = await newGit();
-    await branchWithPj({ dir, git }, '0.1.0-alpha', 'main');
+    await branchWithPj({ dir, git }, '0.1.0-rc', 'main');
 
     await check(dir, {
-      branchState: 'main',
-      isLatestReleaseBranch: false,
+      branchState: 'releaseCandidate',
+      isLatest: true,
       currentBranch: 'main',
       branchType: 'main',
       version: {
         major: 0,
         minor: 1,
         patch: 0,
-        preRelease: 'alpha'
+        preRelease: 'rc'
+      },
+      versionString: '0.1.0-rc'
+    });
+  }).timeout(20000);
```

Comment:
Is beehive in danger of these high-timeout tests becoming a pattern that could make CI take too long? Would it be worth doing more atomic / mock-based tests and then just have a few that take >10s?

Reply:
Yeah, there's a danger of this. It's a tricky balance between real-world end-to-end tests and performant unit tests.

I think StatusTest is a good candidate for mocks. 

Tests are currently at 1min 30s. It's not a huge problem yet, but may need to be addressed in the future.